components/Layout/NextChakraLink.tsx

```diff
@@ -0,0 +1,14 @@
+import { PropsWithChildren } from 'react'
+import NextLink from 'next/link'
+import { LinkProps as NextLinkProps } from 'next/dist/client/link'
+import { Link as ChakraLink, LinkProps as ChakraLinkProps } from '@chakra-ui/core'
+
+export type NextChakraLinkProps = PropsWithChildren<NextLinkProps & Omit<ChakraLinkProps, 'as'>>
+
+export const NextChakraLink = ({ href, as, replace, scroll, shallow, prefetch, children, ...chakraProps }: NextChakraLinkProps) => {
+  return (
+    <NextLink passHref={true} href={href} as={as} replace={replace} scroll={scroll} shallow={shallow} prefetch={prefetch}>
```

Comment:
```suggestion
    <NextLink passHref href={href} as={as} replace={replace} scroll={scroll} shallow={shallow} prefetch={prefetch}>
```

Kalau `true` gak usah dikasih isi prop-nya. Kalau pake atribut doang dia akan default ke `true`.

Reply:
Ini udh di update mas di commit terbaru